packages/client/auth-core/src/utils/jwt.ts

```diff
@@ -1,20 +1,20 @@
-import { CrossmintServiceFactory, type CrossmintServiceWithToken } from "@/services/CrossmintService";
+import { CrossmintAuthService } from "@/services/CrossmintAuthService";
 import { JsonWebTokenError, TokenExpiredError, verify } from "jsonwebtoken";
 
 import { getPublicKey } from "./tokenAuth/publicKey";
 
 export async function verifyCrossmintSessionToken(apiKey: string, token: string) {
-    const crossmintService = CrossmintServiceFactory.create(apiKey, token);
+    const crossmintService = new CrossmintAuthService(apiKey);
```

Comment:
Since the JWT token is mutable I don't think it makes sense for it to be a constructor parameter